lib/src/boolean/signed_area.rs

```diff
@@ -1,12 +1,24 @@
 use geo_types::Coordinate;
 use num_traits::Float;
+use robust::{Coord, orient2d};
 
 #[inline]
 pub fn signed_area<F>(p0: Coordinate<F>, p1: Coordinate<F>, p2: Coordinate<F>) -> F
 where
     F: Float,
 {
-    (p0.x - p2.x) * (p1.y - p2.y) - (p1.x - p2.x) * (p0.y - p2.y)
+    let res = orient2d(
+        Coord{x: p0.x.to_f64().unwrap(), y: p0.y.to_f64().unwrap()},
+        Coord{x: p1.x.to_f64().unwrap(), y: p1.y.to_f64().unwrap()},
+        Coord{x: p2.x.to_f64().unwrap(), y: p2.y.to_f64().unwrap()},
+    );
+    if res > 0. {
+        F::from(1.).unwrap()
+    } else if res < 0. {
+        F::from(-1.).unwrap()
+    } else {
+        F::from(0.).unwrap()
+    }
 }
 
```

Comment:
Actually I'd write this part more like this:

```rust
#[inline]
pub fn coordinate_to_robust<F>(p : Coordinate<F>) -> Coord 
where
    F: Float,
{
    Coord{x: p.x.to_f64().unwrap(), y: p.y.to_f64().unwrap()}
}

#[inline]
pub fn signed_area<F>(p0: Coordinate<F>, p1: Coordinate<F>, p2: Coordinate<F>) -> F
where
    F: Float,
{
    let res = orient2d(
        coordinate_to_robust(p0),
        coordinate_to_robust(p1),
        coordinate_to_robust(p2),
    );
    if res > 0f64 {
        F::one()
    } else if res < 0f64 {
        -F::one()
    } else {
        F::zero()
    }
}
```

... but that is more like a personal preference

Reply:
Looks good to me, I have quickly pushed all the changes...